### PR TITLE
Slick Integration

### PR DESCRIPTION
**Pull request by saadali5997**
I have added model and `repository` for `slick`. Evolution is still not working for some reason, but basic structure is up.

### CODE REVIEW

File: app/repositroy/UserRepository.scala [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    def email = column[String]("email")
+
+    // mapped to Users > password in database
+    def password = column[String]("password")
+
+    // mapped to Users > userRole in database
+    def userRole = column[String]("userRole")
+
+    // mapped to Users > created at in database
+    def authenticated = column[Boolean]("authenticated")
```
Comment (fahadsiddiqui) [inline app/repositroy/UserRepository.scala]: Why do we have this field over here?

File: conf/evolutions.default/1.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+id INTEGER PRIMARY KEY AUTOINCREMENT,
+firstName varchar(255) NOT NULL,
+lastName varchar(255) NOT NULL,
+email varchar(255) NOT NULL UNIQUE,
+password varchar(255) NOT NULL,
+userRole varchar(255) NOT NULL,
+authenticated boolean
+);
+
+INSERT INTO Users(LastName, FirstName, Password, Email, userRole, authenticated) VALUES ('Saad', 'Ali', 'spadsdditcom', 'asdasdasd', 'USER' ,1);
```
Comment (fahadsiddiqui) [inline conf/evolutions.default/1.sql]: Passwords don't go into the table directly, hash them and use them that way.

**Review by fahadsiddiqui — CHANGES_REQUESTED** [review]

File: app/repositroy/UserRepository.scala (hunk 1 above)
Comment (saadali5997) [reply app/repositroy/UserRepository.scala]: For external email verification.

File: conf/evolutions.default/1.sql (hunk 2 above)
Comment (saadali5997) [reply conf/evolutions.default/1.sql]: The `password` string will be hashed string, if that's what you mean

Comment (fahadsiddiqui) [reply conf/evolutions.default/1.sql]: ```LastName, FirstName, ... ```
The case of these arguments doesn't match with table columns.

**Pull request closed without merging**